contributing.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+Welcome to the Pelican project! We are excited to have you contribute to our open-source project. This guide will help you get started with setting up your development environment, understanding our coding standards, and making your first or next contribution.
+
+## Getting started
+
+To start contributing to Pelican Panel, you need to have a basic understanding of the following:
+
+* [PHP](https://php.net) & [Laravel](https://laravel.com)
+* [Livewire](https://laravel-livewire.com) & [Filament](https://filamentphp.com)
+* [Git](https://git-scm.com) & [Github](https://github.com)
+
+## Dev Environment Setup
+
+1. Fork the Repository
+2. Clone your Fork
+3. Install Dependencies (PHP modules & composer, and run `composer install`)
+4. Configure your Environment (via `php artisan p:environment:setup`)
+5. Set up your Database (via `php artisan p:environment:database`) and run Migrations (via `php artisan migrate --seed --force`)
+6. Create your first Admin User (via `php artisan p:user:make`)
+7. Start your Webserver (e.g. Nginx or Apache)
```

Comment:
```suggestion
4. Set file permissions
4.1. Allow write (via `chmod 755 -R storage/* bootstrap/cache`)
4.2. Set .gitignores as read only (via `chmod 644 {bootstrap/cache,storage/framework/testing}/.gitignore`)
5. Configure your Environment (via `php artisan p:environment:setup`)
6. Set up your Database (via `php artisan p:environment:database`) and run Migrations (via `php artisan migrate --seed --force`)
7. Create your first Admin User (via `php artisan p:user:make`)
8. Start your Webserver (e.g. Nginx, Apache or Caddy)
```

Reply:
Setting file permissions is only needed on Linux. The guide mainly targets Windows/ macOS users.